### DISCUSSION TITLE
Repeater - Downstream Interface Not Found

### DISCUSSION THREAD

**Discussion by loyukfai** [Mygod/VPNHotspot#374] [Q&A]
Debug log: https://zerobin.net/?174e606f2d3d6dd3#3Dvv520b8Wmv9AuC3YwfPNdtpL5nBHWRHPU44ha/oHw=

**Comment by Mygod** [comment]
Wi-Fi direct might not be properly supported by your device. Report to your ROM developer.

**Reply by loyukfai** [reply]
But WFD works so I think Wi-Fi direct is ok?

Interestingly, on the first try to enable repeater mode just after boot, I can see the Wi-Fi network name and connect to it, however Windows says "no internet connection"... Also, I cannot turn it off. The only way I can turn it off is to enable Hotspot. 

And if I enable Hotspot, I cannot turn Repeater on again, and it complains about downstream not found...

**Reply by Mygod** [reply]
In that case, try turning off Wi-Fi and on again.

**Reply by loyukfai** [reply]
> In that case, try turning off Wi-Fi and on again.

Ummm.... doesn't seem to help. Do you need more logs?

**Reply by Mygod** [reply]
Okay. Then reboot, turn on repeater, and then export debug information.

**Reply by loyukfai** [reply]
> Okay. Then reboot, turn on repeater, and then export debug information.

https://zerobin.net/?5d175e8ea797c9dc#NNsS47mG/vcrU4CoCdA5XETPKwsTloKSTBQa1duLSl4=

Thanks!

**Reply by loyukfai** [reply]
Was the new log of any help?

Cheers.

**Reply by Mygod** [reply]
Sorry I forgot to check. Post again?

**Reply by loyukfai** [reply]
> Sorry I forgot to check. Post again?

Sure here https://zerobin.net/?74adcc05d4c463f6#8edT0VPfhx0RUTP8BoawiAdUBGvoMY1bPnrbVZD4qG8= sorry for the short expiry date last time

**Reply by Mygod** [reply]
You did not turn on service. Turn on service then export.

**Reply by loyukfai** [reply]
Think I did... Anyhow, here is another try...

https://zerobin.net/?f1d6842a46209e3c#XZ/e7Ze33L+Sp1/Jk9nHxcn4H9t40uvO4RxTS3dSXlI=

I pressed the repeat button, button greyed out, notifications shows repeater enabled, I captured log. Please note the button is unresponsive afterwards. 

Thanks a lot!

**Reply by Mygod** [reply]
Then as I suspected your ROM does not support this properly.

**Reply by loyukfai** [reply]
Ahhhhh... Is there anything I can report to the ROM maintainer to fix it?

**Reply by Mygod** [reply]
That this app and/or Wi-Fi direct is not working properly.

**Reply by loyukfai** [reply]
But Miracast works so I think WFD is not completely not working?
